app/src/test/java/comcmput301f16t01/github/carrier/AcceptingTests.java

```diff
@@ -0,0 +1,91 @@
+     */
+    @Test
+    public void driverAccept(){
+        Rider rider = new Rider("Mike");
+        Request request = new Request(rider, new Location(), new Location());
+        RequestController rc = new RequestController();
+        rc.addRequest(request);
+        Driver driver = new Driver("Cole");
+        assertTrue("Get open request is not returning any open requests", rc.getOpenRequests().size() == 1);
+        rc.addDriver(request, driver);
+        assertTrue("The request is being updated when it shouldn't", rc.getOpenRequests().size() == 1);
+        assertTrue("The request has not been marked as accepted by the driver.", rc.getPendingRequests(driver).size() == 1);
+
+
+
+
+    }
+
+    /**
+     * (US 05.02.01) As a driver, I want to view a list of things I have accepted that are pending, each request with its description, and locations.
+     */
+    @Test
+    public void driverPendingAccepts(){
+        Rider rider = new Rider("Mike");
+        Request request = new Request(rider, new Location(), new Location());
+        RequestController rc = new RequestController();
+        rc.addRequest(request);
+        Driver driver = new Driver("Cole");
+        assertTrue("Get open request is not returning any open requests", rc.getOpenRequests().size() == 1);
+        rc.addDriver(request, driver);
+        // Add another driver to compare
+        Driver driver1 = new Driver("WASD");
+        Request request1 = new Request(rider, new Location(), new Location());
+        rc.addRequest(request1);
+        rc.addDriver(request1,driver1);
+        assertFalse("Two drivers share pending requests when they shouldn't", rc.getPendingRequests(driver).equals(rc.getPendingRequests(driver1)));
+        assertTrue("The request has not been marked as accepted by the driver.", request.getAcceptedDrivers().contains(driver));
+        assertTrue("Driver is not able to view their pending requests .", rc.getPendingRequests(driver).size() == 1);
+        assertTrue("Request in pending requests is not the same.", rc.getPendingRequests(driver).get(0).equals(request));
+        assertEquals("The request is not being updated to pending", Request.ACCEPTED, request.getStatus());
+
+    }
+    /**
+     * (US 05.03.01) As a driver, I want to see if my acceptance was accepted.
+     */
+    @Test
+    public void driverAcceptedRequests(){
+        Rider rider = new Rider("Josh");
+        Driver driver = new Driver("Kevin");
+        Request request = new Request(rider, new Location(), new Location());
+        RequestController rc = new RequestController();
+        rc.addRequest(request);
+        rc.addDriver(request, driver);
+        rc.acceptDriver(request, driver);
+        assertTrue("Driver has been added to the request as the confirmed driver", request.getConfirmedDriver().equals(driver));
+        assertFalse("The request is still available to be accepted by other drivers", rc.getAvailableRequests().contains(request));
```

Comment:
I kind of understand what "Available Request" does for us but how is this represented in the Request class? I think we need to define this better. I feel like it is too ambiguous.